.github/workflows/test_typing.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         pip install .[dev,x]
     - name: Run Tests
       shell: bash -l {0}
-      run: pytest -v --cov=kornia --cov-report xml --mypy kornia/ -m mypy
+      run: pytest -v --cov=kornia --cov-report xml --mypy kornia/
```

Comment:
I don't know why, is using a plugin a pytest plugin for mypy instead of just run mypy. Just to have coverage report?

Reply:
Yes, but we can change that based on what improvements the whole ci testing